### ISSUE TITLE
[Nomination] : Frank Dierolf

### ISSUE THREAD

**Issue by FrankBevr** [Polkadot-Heroes/Members#61]
## Hola everyone 👋,

I'm Frank and self nominate myself.

### Lets try to selfscore myself.

Technical Expertise: 7-8
Community Involvement: 5-6
Content Creation Experience: 3-4
Leadership Experience: 3-4

### Lets write sentences to each topic

**Technical Expertise**
- Rust 9 months ago, Typescript 2.5years ago , 3D/C#  5 years ago  
- [SwipePM - Hack](https://github.com/FrankBevr/SwipePM) -  Last related polka project project 

### Community involvement

**Community Involvement**
- +10 in Person Polka Events  &  +5 Polka Munich Meetups 

### Content creation

**Content Creation Experience**
- Hackathons could count as content

### Leadership experience

**Leadership experience**
- Maybe Berlin Blockchain Week 2022 - Polka Event

### Mentorship experience

Yes

### Lets write my sentence:

My natural strong suite are people & ideas.  
My generall weak spot is organizing.
Doing 1 o 1 mentoring. Sounds fun. I mostly do it anyways.
Speak at Events sounds fun as well. Usually I'm not the worst person for it.
I dont look for such things, if a being a hero would elvate the request rate, sure why not.

Basically, I'm Frank, doing Web3 Development in Polkadot. I like the community, found some friends there and so I'm quite involved. Reasonable confident about my expertise. I like people, so just write me if questions or just chatting about the latest stuff🦆

## Cheers ✌️

**Comment by zsofimajor** [comment]
Hi @FrankBevr, thank you again for submitting your nomination to become a Hero in the Polkadot Developer Heroes program. 
We would like to inform you of an important update regarding the approval criteria for our program. After some careful consideration,  we've decided to update our criteria to minimise subjectivity as much as possible  and ensure fairness and transparency in the Hero selection process. As a result, we kindly ask that you resubmit your application following the new criteria, which you can find [here](https://github.com/Polkadot-Heroes/Members/blob/main/Heroes_Review_Criteria.md). 

Your initial application showed us your passion and dedication, and we'd love for you to build upon that. Once again, thanks a ton for your interest in the Heroes program. We're looking forward to receiving your updated application. Your support means a lot to us, and we're grateful for your understanding as we make these changes.

**Comment by FrankBevr** [comment]
# Resubmission - Frank Dierolf

## Scoring Framework:

### Technical Expertise

<details>
<summary>Original</summary>

| GitHub activity related to Polkadot - in the past quarter                                                                                                                                                         | Links          | Points to earn |
| :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------- | :------------- |
| Is an owner or maintainer of any other open-source Polkadot ecosystem project <br>_(where the project is not the Polkadot SDK, not a Substrate based chain, not a tooling <br>or library project, or not a dapp)_ | GitHub link(s) | 3              |

| Technical Education - in the past quarter                                                                            | Scoring via                    | Points to earn                                                |
| :------------------------------------------------------------------------------------------------------------------- | :----------------------------- | :------------------------------------------------------------ |
| Has taught a coding workshop or training session on Polkadot <br>and/or Rust in the past quarter                     | Event link(s)                  | 1 session = 1 pt<br>2 sessions = 2 pts<br>3+ sessions = 3 pts |
| Has contributed more than once to documentation for Polkadot <br>or a Polkadot ecosystem project in the past quarter | GitHub link(s)                 | 2                                                             |
| Has published a technical research paper or article <br>about Rust or Polkadot with more than 60% of own content     | Content link(s)                | 2                                                             |
| Is in the top 10% on StackExchange                                                                                   | StackExchange <br>profile link | 1                                                             |
| Has been graduated from the Polkadot Blockchain Academy                                                              | Year - Location                | 2                                                             |

</details>

<details>
<summary>Self Evalution</summary>

| GitHub activity related to Polkadot - in the past quarter                     | Links                                                                                                                                                                                                    | Points |
| :---------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :----- |
| Is an owner or maintainer of any other open-source Polkadot ecosystem project | [GH - SwipePM - Previous Hack](https://github.com/frankbevr/swipepm), [GH - Inkora - Current Hack](https://github.com/frankbevr/inkora), [GH - Key3d - Current Hack](https://github.com/frankbevr/key3d) | 3      |

| Technical Education - in the past quarter                                                                            | Scoring via                    | Points to earn |
| :------------------------------------------------------------------------------------------------------------------- | :----------------------------- | :------------- |
| Has taught a coding workshop or training session on Polkadot <br>and/or Rust in the past quarter                     | Event link(s)                  | 0              |
| Has contributed more than once to documentation for Polkadot <br>or a Polkadot ecosystem project in the past quarter | GitHub link(s)                 | 0              |
| Has published a technical research paper or article <br>about Rust or Polkadot with more than 60% of own content     | Content link(s)                | 0              |
| Is in the top 10% on StackExchange                                                                                   | StackExchange <br>profile link | 0              |
| Has been graduated from the Polkadot Blockchain Academy                                                              | Year - Location                | 0              |

</details>

**Points: 3/13**

### Community involvement

<details>
<summary>Original</summary>

| Activity on the Polkadot Forum - in the past quarter                       | Scoring via        | Points to earn |
| :------------------------------------------------------------------------- | :----------------- | :------------- |
| Has authored more than one topic on the forum in the past quarter          | Forum profile link | 1              |
| Has added more than one comment to topics on the forum in the past quarter | Forum profile link | 1              |

| Polkadot forum technical contribution - Tech talk category                                                                                                                                    | Scoring via        | Points to earn |
| :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :----------------- | :------------- |
| Created a technical topic for discussion in the tech talk section of the forum in the past quarter <br> _(Non tech topics can still be submitted in this category, but will not count here.)_ | Forum profile link | 1              |
| Gave technical responses to 3 or more topics in the tech talk section of the forum in the past quarter                                                                                        | Forum profile link | 1              |

| Attendance at Polkadot-related events - in the past quarter                                                                                               | Scoring via   | Points to earn |
| :-------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------ | :------------- |
| Has attended two or more Polkadot-related events in the past quarter <br>_(e.g. meetups, other community events that's not sub0, Decoded or a hackathon)_ | Event link(s) | 1              |
| Has attended most recent sub0                                                                                                                             | Event link(s) | 1              |
| Has attended most recent Decoded                                                                                                                          | Event link(s) | 1              |

| Participation at a Polkadot hackathon (Polkadot-focused or has a Polkadot track) | Scoring via               | Points to earn |
| :------------------------------------------------------------------------------- | :------------------------ | :------------- |
| Has participated at a hackathon with a submitted project in the past quarter     | Event and project link(s) | 1              |
| Won a hackathon with a submitted project in the past quarter                     | Event and project link(s) | 1              |

| Mentoring within the Heroes program                  | Scoring via | Points to earn                                                  |
| :--------------------------------------------------- | :---------- | :-------------------------------------------------------------- |
| Has been mentoring folk(s) within the Heroes program | Name(s)     | 1 mentored = 1 pt<br>2 mentored = 2 pts <br>3+ mentored = 3 pts |

| Polkadot Ambassador Program                                | Scoring via | Points to earn |
| :--------------------------------------------------------- | :---------- | :------------- |
| Is a Polkadot Ambassador/Senior Ambassador/Head Ambassador | Name        | 1              |

| Community moderation                                                                                   | Scoring via | Points to earn |
| :----------------------------------------------------------------------------------------------------- | :---------- | :------------- |
| Moderator or active contributor in Polkadot-focused online communities <br>_(e.g., Telegram, Discord)_ | Name        | 1              |

</details>

<details>
<summary>Self Evalution</summary>

| Activity on the Polkadot Forum - in the past quarter                       | Scoring via        | Points to earn |
| :------------------------------------------------------------------------- | :----------------- | :------------- |
| Has authored more than one topic on the forum in the past quarter          | Forum profile link | 0              |
| Has added more than one comment to topics on the forum in the past quarter | Forum profile link | 0              |

| Polkadot forum technical contribution - Tech talk category                                                                                                                                    | Scoring via        | Points to earn |
| :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :----------------- | :------------- |
| Created a technical topic for discussion in the tech talk section of the forum in the past quarter <br> _(Non tech topics can still be submitted in this category, but will not count here.)_ | Forum profile link | 0              |
| Gave technical responses to 3 or more topics in the tech talk section of the forum in the past quarter                                                                                        | Forum profile link | 0              |

| Attendance at Polkadot-related events - in the past quarter                                                                                               | Scoring via                                                            | Points to earn |
| :-------------------------------------------------------------------------------------------------------------------------------------------------------- | :--------------------------------------------------------------------- | :------------- |
| Has attended two or more Polkadot-related events in the past quarter <br>_(e.g. meetups, other community events that's not sub0, Decoded or a hackathon)_ | [Sub0](https://www.polkadot.network/ecosystem/events/sub0/)            | 1              |
| Has attended most recent sub0                                                                                                                             | [Sub0](https://www.polkadot.network/ecosystem/events/sub0/)            | 1              |
| Has attended most recent Decoded                                                                                                                          | [Decoded](https://www.polkadot.network/ecosystem/events/decoded-2023/) | 1              |

| Participation at a Polkadot hackathon (Polkadot-focused or has a Polkadot track) | Scoring via                                                                  | Points to earn |
| :------------------------------------------------------------------------------- | :--------------------------------------------------------------------------- | :------------- |
| Has participated at a hackathon with a submitted project in the past quarter     | [Angel Hack - Polka Global](https://www.polkadotglobalseries.com/apac-2023/) | 1              |
| Won a hackathon with a submitted project in the past quarter                     | [2n Price could count^^ - SwipePM](https://github.com/Frankbevr/SwipePM)     | 1              |

| Mentoring within the Heroes program                  | Scoring via                                                      | Points to earn |
| :--------------------------------------------------- | :--------------------------------------------------------------- | :------------- |
| Has been mentoring folk(s) within the Heroes program | I help out a ambassador friend(write if you wanna know the name) | 1              |

| Polkadot Ambassador Program                                | Scoring via | Points to earn |
| :--------------------------------------------------------- | :---------- | :------------- |
| Is a Polkadot Ambassador/Senior Ambassador/Head Ambassador | Frank       | 1              |

| Community moderation                                                   | Scoring via                                                             | Points to earn |
| :--------------------------------------------------------------------- | :---------------------------------------------------------------------- | :------------- |
| Moderator or active contributor in Polkadot-focused online communities | I don't moderate it, but write messages in KodaDot Discord, tiny server | 1              |

</details>

**Points: 8/15**

### Content creation

<details>
<summary>Original</summary>

| Written content - in the past quarter                                                                         | Scoring via     | Points to earn |
| :------------------------------------------------------------------------------------------------------------ | :-------------- | :------------- |
| Has produced more than one technical blog post                                                                | Content link(s) | 2              |
| Has produced any interactive educational content related to Polkadot <br> _(e.g. coding challenges, quizzes)_ | Content link(s) | 1              |
| Has produced any Polkadot-related educational content in other languages than English                         | Content link(s) | 2              |
| Has created and shared Polkadot-related infographics or visual content in the past quarter                    | Content link(s) | 1              |

| Recorded content - in the past quarter                                                                                   | Scoring via     | Points to earn |
| :----------------------------------------------------------------------------------------------------------------------- | :-------------- | :------------- |
| Produced more than one technical-focused video on own Youtube (or other platform) or podcast channel in the past quarter | Content link(s) | 2              |
| Has been a guest speaker on a Polkadot-focused YouTube (or other platform) channel or podcast in the past quarter        | Content link(s) | 1              |

| Public speaking - in the past quarter                                                                        | Scoring via                  | Points to earn                                           |
| :----------------------------------------------------------------------------------------------------------- | :--------------------------- | :------------------------------------------------------- |
| Served as a speaker or panelist at conferences or events on <br>a Polkadot-related topic in the past quarter | Event (or recording) link(s) | 1 event = 1 pt<br>2 events = 2 pts <br>3+ events = 3 pts |

</details>

<details>
<summary>Self Evalution</summary>
| Written content - in the past quarter                                                      | Scoring via     | Points to earn |
| :----------------------------------------------------------------------------------------- | :-------------- | :------------- |
| Has produced more than one technical blog post                                             | Content link(s) | 0              |
| Has produced any interactive educational content related to Polkadot                       | Content link(s) | 0              |
| Has produced any Polkadot-related educational content in other languages than English      | Content link(s) | 0              |
| Has created and shared Polkadot-related infographics or visual content in the past quarter | Content link(s) | 0              |

| Recorded content - in the past quarter                                                                                   | Scoring via                                                                                                        | Points to earn |
| :----------------------------------------------------------------------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------- | :------------- |
| Produced more than one technical-focused video on own Youtube (or other platform) or podcast channel in the past quarter | If friends ask about the PBA and the Exam I sendt this, its [content](https://www.youtube.com/watch?v=nluMMb2m1vg) | 2              |
| Has been a guest speaker on a Polkadot-focused YouTube (or other platform) channel or podcast in the past quarter        | Content link(s)                                                                                                    | 0              |

| Public speaking - in the past quarter                                                                        | Scoring via                                                                   | Points to earn |
| :----------------------------------------------------------------------------------------------------------- | :---------------------------------------------------------------------------- | :------------- |
| Served as a speaker or panelist at conferences or events on <br>a Polkadot-related topic in the past quarter | [Introduced People on sub0](https://youtu.be/ahtBhzwmjY4?si=5JI1d-e1Q9QOXeaB) | 1              |

</details>

**Points: 3/12**

### Leadership experience

<details>
<summary>Original</summary>

| Organization of or participation at Polkadot-related tech focused events - in the past quarter                         | Scoring via   | Points to earn |
| :--------------------------------------------------------------------------------------------------------------------- | :------------ | :------------- |
| Has organized or hosted a Polkadot related technical community meetup in the past quarter                              | Event link(s) | 2              |
| Has organized or hosted a Polkadot related technical workshop in the past quarter                                      | Event link(s) | 2              |
| Has organized other community initiatives (eg. Polkadot-focused hackathon or <br>coding challenge) in the past quarter | Event link(s) | 2              |
| Served as a judge or mentor at Polkadot hackathons in the past quarter                                                 | Event link(s) | 1              |

| Active involvement in Polkadot based projects                                    | Scoring via                           | Points to earn |
| :------------------------------------------------------------------------------- | :------------------------------------ | :------------- |
| Founded or is leading a Polkadot-related working group or special interest group | Link(s) for more info                 | 2              |
| Is holding an engineering leadership position in a Polkadot-based project        | GitHub or other Link(s) for more info | 2              |

| Leading discussions in the online space                                                                                                                                                                                                            | Scoring via          | Points to earn |
| :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------------- | :------------- |
| Is creating and managing a Polkadot-focused media outlet or publication <br>_(Including discussion starters, with twitter threads, acive discussions started in chat rooms, forums, or productions similar to DotLeap, AAG, any newsletters etc.)_ | Link(s)<br> Username | 1              |

</details>

<details>
<summary>Self Evalution</summary>
| Organization of or participation at Polkadot-related tech focused events - in the past quarter                         | Scoring via                                                   | Points to earn |
| :--------------------------------------------------------------------------------------------------------------------- | :------------------------------------------------------------ | :------------- |
| Has organized or hosted a Polkadot related technical community meetup in the past quarter                              | [Munich Meetups](https://www.youtube.com/watch?v=NtZ-NQSQFrQ) | 2              |
| Has organized or hosted a Polkadot related technical workshop in the past quarter                                      | in person or via google meet to some folks that i met         | 1              |
| Has organized other community initiatives (eg. Polkadot-focused hackathon or <br>coding challenge) in the past quarter | Event link(s)                                                 | 0              |
| Served as a judge or mentor at Polkadot hackathons in the past quarter                                                 | Event link(s)                                                 | 0              |

| Active involvement in Polkadot based projects                                    | Scoring via                           | Note           | Points to earn |
| :------------------------------------------------------------------------------- | :------------------------------------ | :------------- | :------------- |
| Founded or is leading a Polkadot-related working group or special interest group | Link(s) for more info                 | I'm a Founder  | 1              |
| Is holding an engineering leadership position in a Polkadot-based project        | GitHub or other Link(s) for more info | I'm my own CTO | 1              |

| Leading discussions in the online space                                                                                                                                                                                                            | Scoring via          | Points to earn |
| :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------------- | :------------- |
| Is creating and managing a Polkadot-focused media outlet or publication <br>_(Including discussion starters, with twitter threads, acive discussions started in chat rooms, forums, or productions similar to DotLeap, AAG, any newsletters etc.)_ | Link(s)<br> Username | 0              |

</details>

**Points: 5/10**

## Outcome

- Total Points available: 50
- Necessary Points: 25
- Earned Points: 19

Passed? ­🙅

> **Todo for next submission:**
>
> - Write/Record/Comment some stuff, publicly ­🐒😅

­🧑‍🏭 W.I.P. ­ 🧑‍🏭
